### ISSUE TITLE
Filter operator validity error

### ISSUE THREAD

**Issue by honzagula** [Fousky/iDoklad#12]
When I want to user filter with iDokladFilter::GREATER_THEN operator, I get following error:

`Operator gt is not valid. Available operators: lt, lte, gt, gte, eq, !eq, ct, !ct, between`.

My code:

```
$filter = new iDokladFilter();
$filter->filter('DateOfMaturity', iDokladFilter::GREATER_THEN, $from->format('Y-m-d'));
```

**Comment by kockodev** [comment]
@honzagula I'll check this behavior

**Comment by kockodev** [comment]
@honzagula fixed in [0.2.8-dev](https://github.com/Fousky/iDoklad/releases/tag/0.2.8-dev), can you try it?

**Comment by honzagula** [comment]
Its working now. Thank you! But now I have another filter problem. I have created new issue: #13
